api/cloud_api/util/S3BucketApiHandler.js

```diff
@@ -8,9 +8,14 @@ class S3BucketApiHandler {
       secretAccessKey: s3BucketKeys.AWSSECRETKEY,
       region: 'us-west-1',
     });
+    /* eslint-disable-next-line */
+    this.hasValidKeys = (s3BucketKeys.AWSACCESSKEYID != 'NOT_SET' && s3BucketKeys.AWSSECRETKEY != 'NOT_SET') ? true : false;
```

Comment:
how about doing the if statement like the google api handler does?

if (both not set) 
  stuff